k8s.io/README.md

```diff
@@ -3,6 +3,37 @@ Overview
 This contains the Nginx configuration for k8s.io and the associated subdomain
 redirectors.
 
+Vanity URL(s)
+====
+
+|  | k8s.io | kubernetes.io |
+| --- | --- | --- |
+| APT downloads| https://apt.k8s.io | https://apt.kubernetes.io |
+| Changelog | https://changelog.k8s.io | https://changelog.kubernetes.io |
+| CI logs | https://ci-test.k8s.io | https://ci-test.kubernetes.io |
+| Git repo | https://code.k8s.io | https://code.kubernetes.io |
+| Community submit queue | https://community.submit-queue.k8s.io |
+| Contrib submit queue | https://contrib.submit-queue.k8s.io |
+| Downloads XML | https://dl.k8s.io | https://dl.kubernetes.io |
```

Comment:
It shows the s3-style XML if you don't specify a path, though it's meant to be used with a URI like https://dl.k8s.io/v1.7.5/kubernetes.tar.gz.

Maybe just drop the "XML"?

Reply:
Done!